modelforge/potential/models.py

```diff
@@ -90,10 +84,60 @@ def forward(
             - 'd_ij' : torch.Tenso, shape (3, n_pairs)
```

Comment:
minor typo: "torch.Tensor" I also think you swapped the shape of r_ij and d_ij in the doc-string

Reply:
thanks for catching that!